qiskit/circuit/library/standard_gates/__init__.py

```diff
@@ -171,4 +177,15 @@ def get_standard_gate_name_mapping():
         Measure(),
     ]
     name_mapping = {gate.name: gate for gate in gates}
+    # Add variable width gates as a mapping between names and classes
+    # since an instance requires knowing how many qubits are being operated on
+    name_mapping["if_else"] = IFElseOp
```

Comment:
typo here (extra capital letter)

Reply:
Sigh, yeah and I missed an op below

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/8935/commits/210553e8fbab53130255c8bcdf2db9f42b31c87a